mitosheet/esbuild.dash.js

```diff
@@ -0,0 +1,56 @@
+const esbuild = require('esbuild');
+const react = require('react');
+
+
+/**
+ * This plugin is used to make react external to the bundle in the mitodashwrapper.js file.
+ * 
+ * This is necessary as we want to use the react that is already loaded in the page for dash, 
+ * and is defined globally as `window.React`.
```

Comment:
Is this the solution to the problem you had where you couldn't separate Mito Frontend into a separate npm package because they would have different versions of react from Jupyter?